sys/include/net/netopt.h

```diff
@@ -803,6 +806,30 @@ typedef enum {
      */
     NETOPT_RSSI,
 
+    /**
+     * @brief   (@ref netopt_scan_request_t) Instruct the interface to do a network scan
+     *
+     * This netopt triggers an asynchronous network scan.
+     * The result is a list of reachable access points @ref l2scan_list_t.
+     * Notification happens by a callback @ref netopt_on_scan_result_t.
+     */
+    NETOPT_SCAN,
```

Comment:
Are these options (`SCAN`, `CONNECT` and `DISCONNECT`) unique for WiFi? If yes, I wonder whether we should call them `NEOPT_WIFI_*`.

Reply:
I also see them for 802.15.4 when a node wants to connect to a PAN coordinator

Reply:
I am not sure if there is active scanning in 802.15.4 but receiving of beacons exist at least